components/empty-state/src/empty-state.ts

```diff
@@ -186,7 +190,11 @@ export class DtEmptyState
   constructor(
     private _changeDetectorRef: ChangeDetectorRef,
     private _elementRef: ElementRef<HTMLElement>,
-    private _viewportResizer: DtViewportResizer,
+    /**
+     * @deprecated Remove the viewportResizer from the constructor as it is no longer needed.
+     * @breaking-change Remove the viewportResizer in version 8.0.0
```

Comment:
can we do this with version 7? :D

Reply:
I believe that at some point we have discussed this in the past and decided that we will announce breaking-changes 2 major versions in advance.